docs/content/entity-checkout-process.md

```diff
@@ -59,29 +59,30 @@ To perform more complex setup logic, you can add components that themselves act
 
 ### Representing your Entity with a GameObject
```

Comment:
in general, this doc feels like enough has changed that we should use our standard docs procedure: 
use a google docs to make it easier for us and the tech writers to review this

Reply:
Tbh I'd rather just update this reasonably quickly especially because the main documentation would be in a bunch of new pages we write just for monobehaviours, which will happen through google docs etc. These changes are just to make sure the old docs are not actually incorrect. Do you think that makes it OK or should I still do it via a Google Doc? (Actually I should probably ask that from Laura. I did send her a link to these commits, I'll follow up with that question)

Reply:
Plan is to link to a GDoc with WIP in this MD file with the note:

`We are currently updating this documentation - see the work in progress in a [Google document](link).`

https://docs.google.com/document/d/e/2PACX-1vQxPlFX6BPaArx4BpOHRHDHAW_iarJ8Dqd7-6U6na-3s44Ty_0YTtZELmpIViXSvD0KqzKKl7V1TIUD/pub

(You could put it in a table if you want to make it pretty - and draw attention to it!?)